src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -422,7 +425,7 @@ private static Job MakeWasmJob(Job baseJob, CommandLineOptions options, TimeSpan
             bool wasmAot = options.AOTCompilerMode == MonoAotCompilerMode.wasm;
 
             var wasmRuntime = new WasmRuntime(
-                mainJs: options.WasmMainJs,
+                mainJs: options.WasmMainJs ?? new FileInfo(options.RuntimeSrcDir.ToString() + @"\src\mono\wasm\runtime-test.js"),
```

Comment:
It would be better to use `Path.Combine`

```suggestion
                mainJs: options.WasmMainJs ?? new FileInfo(Path.Combine(options.RuntimeSrcDir.FullName, "src", "mono", "wasm", "runtime-test.js")),
```